assembl/graphql/schema.py

```diff
@@ -480,13 +511,17 @@ def resolve_order(self, args, context, info):
 class Query(graphene.ObjectType):
     node = Node.Field()
     posts = SQLAlchemyConnectionField(PostConnection, idea_id=graphene.ID())
-    ideas = SQLAlchemyConnectionField(Idea)
+    root_idea = graphene.Field(Idea)
```

Comment:
I implemented a root_idea query. It may be a good idea with the proposed UI to actually get the ideas for a given level when you actually need it. For the first level, do a `rootIdea { children { title } }`. For level 2, so the children ideas of a given idea `node(id:$ideaId) { children { title } }` and so on...